web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/CertificateSignConfirmationFunctionImpl.java

```diff
@@ -50,13 +53,15 @@ public CertificateSignConfirmationFunctionImpl(UtkastService utkastService,
 
     @Override
     public Optional<ResourceLinkDTO> get(Certificate certificate, WebCertUser webCertUser) {
-        if (!authoritiesHelper.isFeatureActive(AuthoritiesConstants.FEATURE_UNIKT_INTYG, certificate.getMetadata().getType())) {
+        if (isUniqueFeatureActive(certificate)) {
```

Comment:
Maybe need to rename the method or move the "negating" operand (!) to outside the method. Now it read like if the unique feature is active it shouldn't return any resource link.

Reply:
I agree. Changed to negating operande.